files/familysearch-javascript-sdk/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "familysearch-javascript-sdk",
+  "repo": "FamilySearch/familysearch-javascript-sdk",
+  "files": {
+    "include": ["*.js"]
```

Comment:
This wont work. There is no `familysearch-javascript-sdk.min.js` in here https://github.com/FamilySearch/familysearch-javascript-sdk/tree/v1.0.0-beta